packages/snaps-sdk/src/jsx/components/form/AccountSelector.ts

```diff
@@ -0,0 +1,59 @@
+import type { CaipAccountId, CaipChainId } from '@metamask/utils';
+
+import { createSnapComponent } from '../../component';
+
+/**
+ * The props of the {@link AccountSelector} component.
+ *
+ * @property name - The name of the account selector. This is used to identify the
+ * state in the form data.
+ * @property hideExternalAccounts - Whether to hide accounts that don't belong to the snap.
+ * @property chainIds - The chain IDs to filter the accounts to show.
+ * @property switchGlobalAccount - Whether to switch the selected account in the client.
+ * @property value - The selected address.
+ */
+export type AccountSelectorProps = {
+  name: string;
+  hideExternalAccounts?: boolean | undefined;
+  chainIds?: CaipChainId[] | undefined;
+  switchGlobalAccount?: boolean | undefined;
```

Comment:
Is this a requirement for the first iteration of this component?

Reply:
Yes it is, it's needed to show the correct account when coming back to the client home page :)